src/pages/user/User.tsx

```diff
@@ -57,8 +57,8 @@ const User: React.FC = () => {
   const dispatch = useDispatch();
   const history = useHistory();
 
-  const hasHandles = user.authority?.handles?.length > 0;
-  const hasFeide = user.authority?.feideids?.length > 0;
+  const hasHandles = user.authority && user.authority.handles?.length > 0;
+  const hasFeide = user.authority && user.authority?.feideids?.length > 0;
```

Comment:
noen grunn til at du endret på dette?

Reply:
length av mulig null virker dårlig

Reply:
ahh. men trenger uansett ikke det ekstra spørsmålstegnet bak authority når du sjekker for det først da